test/test_corpus.cc

```diff
@@ -165,10 +165,6 @@ TEST_P(ExpectationTest, PerPhaseTest) { // NOLINT
     vector<core::ErrorRegion> errs;
     for (auto file : files) {
         errs.emplace_back(gs, file);
-        if (file.data(ctx).source().find("# typed:") == string::npos) {
```

Comment:
@jvilk-stripe this was intentional. Why did you remove it?

Reply:
We want all test files to be explicit about what `typed` level they want to be in.

Reply:
@DarkDimius without removing this line, I cannot test this bug...

Reply:
Unless I make `# typed: none` a valid type sigil?

Reply:
Can we hardcode name of the file? we already have prior art for this
https://github.com/sorbet/sorbet/blob/7bbc8c652d4c1ea044d01d59778beea3a5627fb1/core/Context.cc#L34